### ISSUE TITLE
TimeoutError, silent fail of play to use

### ISSUE THREAD

**Issue by starcraftman** [starcraftman/diceBot#16]
Annoying problem due to wanting to keep player largely non-async. 
Fetch code is async, player.play is not, hard to wait on the future download.

Either fix this quickly or see quickly about streaming possibility.

**Comment by starcraftman** [comment]
Should be fixed, though root problem remains I only remedied the symptom.

**Comment by starcraftman** [comment]
Had to do with the player monitor and pruning cache with files not set to recent c/atime.

Need to make a mechanism to redownload files deleted or preferably at this point I may just look at streaming while I can. Bug is mostly fixed but may recur as root is filled cache + no redownload.

**Comment by starcraftman** [comment]
To ultimately fix this, I should probably modify prune_cache to issue a modification to live player and delist the video if it gets removed.

__Required Changes__
- Prioritize deleting songs not in current player queue.
- Ability to match current queue songs to ones left in cache (free function?).
- First delete any songs not in current queue then prune based on last played time since all current will have updated times.

**Comment by starcraftman** [comment]
No longer relevant, either a file is local or it is remote and streamed. No more caching of remote songs.
